### DISCUSSION TITLE
有没有解决国内无法访问vercel.app的方法？

### DISCUSSION THREAD

**Discussion by lfyszy** [Qexo/Qexo#438] [Q&A]
有没有解决国内无法访问vercel.app的方法？国内无法打开

**Comment by Fgaoxing** [comment]
改hosts



在 2023-08-24 15:46:30，"lfyszy" ***@***.***> 写道：

有没有解决国内无法访问vercel.app的方法？国内无法打开

**Reply by lfyszy** [reply]
请问一下如何改呢

**Comment by Fgaoxing** [comment]
baidu



在 2023-08-24 20:12:22，"lfyszy" ***@***.***> 写道：

请问一下如何改呢

**Reply by lfyszy** [reply]
谢谢！
